README.md

```diff
@@ -71,22 +71,23 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+The helper method because it helps make a controller method  available to the view.
 ```
```

Comment:
:+1: true, it is best defined as a helper method. But a better way of thinking about it is that helper methods influence how information is displayed in the browser (i.e., how HTML is rendered).